server/src/main.rs

```diff
@@ -45,13 +51,38 @@ pub struct Context {
 
 /** Public Functions **/
 fn main() {
+    
+
```

Comment:
There should not be any newlines in the start of a function.

Reply:
Fixed